.github/workflows/terraform-deployment.yml

```diff
@@ -0,0 +1,62 @@
+name: Terraform deployment
+
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  plan:
+    name: Terraform plan
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup terraform
+      uses: hashicorp/setup-terraform@v1
+      with:
+        terraform_version: 0.15.0
+
+    - name: Download backend zip
+      run: curl -sLO https://github.com/fap-cloud-service/fap-backend/releases/latest/download/friends-and-places-server.zip
+
+    - name: Terraform init and plan
+      env:
+        ARM_CLIENT_ID: ${{ secrets.ARM_CLIENT_ID }}
+        ARM_CLIENT_SECRET: ${{ secrets.ARM_CLIENT_SECRET }}
+        ARM_TENANT_ID: ${{ secrets.ARM_TENANT_ID }}
+        ARM_SUBSCRIPTION_ID: ${{ secrets.ARM_SUBSCRIPTION_ID }}
+      run: |
+        terraform init
+        terraform plan -input=false
```

Comment:
Nice to have:
Den Plan mit dem Parameter `-out=plan.tfplan` in eine Datei speichern und als Artefakt mit dem zweiten Job teilen. Ansonsten können die Änderungen die applyed werden von dem ausgegebenen Plan abweichen. Das erschwert ggf. debugging

Reply:
Stimmt, den Hinweis hatte ich bereits im Log gesehen.
Follow-Up Commit mit Fix kommt.

Reply:
Done.